client/app/components/accounts/server.cjsx

```diff
@@ -118,3 +111,13 @@ module.exports = AccountServer = React.createClass
         _.each state, (value, key) ->
             globalState["#{protocol}#{key[0].toUpperCase()}#{key.slice 1}"] = value
         @props.onChange globalState
+
+
+    _securityOptions: ->
```

Comment:
Why are you using `_`as a prefix? I think that if you really want this method to be private, you should remove it from ReactComponent or do nothing else than removing the prefix :)